proto/regen/ecocredit/v1/state.proto

```diff
@@ -10,224 +10,232 @@ option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/core";
+  bytes address = 1;
 
-    // batch_id is the id of the credit batch
-    uint64 batch_id = 2;
+  // batch_id is the id of the credit batch
+  uint64 batch_id = 2;
 
-    // tradable is the tradable amount of credits
-    string tradable = 3;
+  // tradable is the tradable amount of credits
+  string tradable = 3;
 
-    // retired is the retired amount of credits
-    string retired = 4;
+  // retired is the retired amount of credits
+  string retired = 4;
 
-    // escrowed is the amount of credits locked up in escrow for the marketplace.
-    string escrowed = 5;
+  // escrowed is the amount of credits locked up in escrow for the marketplace.
+  string escrowed = 5;
 }
 
 // BatchSupply tracks the supply of a credit batch
 message BatchSupply {
-    option (cosmos.orm.v1alpha1.table) = {
-        id: 10,
-        primary_key: {fields: "batch_id"}
-    };
-
-    // batch_id is the id of the batch
-    uint64 batch_id = 1;
-
-    // tradable_amount is the total number of tradable credits in the credit batch.
-    // Some of the issued credits may be cancelled and will be removed from
-    // tradable_amount and tracked in amount_cancelled. tradable_amount + retired_amount +
-    // amount_cancelled will always sum to the original credit issuance amount.
-    string tradable_amount = 2;
-
-    // retired_amount is the total amount of credits that have been retired from the credit batch.
-    string retired_amount = 3;
-
-    // cancelled_amount is the number of credits in the batch that have been
-    // cancelled, effectively undoing the issuance. The sum of total_amount and
-    // amount_cancelled will always equal the original credit issuance amount.
-    string cancelled_amount = 4;
-
-    // escrowed_amount is the amount of credits in the batch that have been
-    // locked up in escrow for use in the marketplace.
-    string escrowed_amount = 5;
+  option (cosmos.orm.v1alpha1.table) = {
+    id : 10,
+    primary_key : {fields : "batch_id"}
+  };
+
+  // batch_id is the id of the batch
+  uint64 batch_id = 1;
+
+  // tradable_amount is the total number of tradable credits in the credit
+  // batch. Some of the issued credits may be cancelled and will be removed from
+  // tradable_amount and tracked in amount_cancelled. tradable_amount +
+  // retired_amount + amount_cancelled will always sum to the original credit
+  // issuance amount.
+  string tradable_amount = 2;
+
+  // retired_amount is the total amount of credits that have been retired from
+  // the credit batch.
+  string retired_amount = 3;
+
+  // cancelled_amount is the number of credits in the batch that have been
+  // cancelled, effectively undoing the issuance. The sum of total_amount and
+  // amount_cancelled will always equal the original credit issuance amount.
+  string cancelled_amount = 4;
+
+  // escrowed_amount is the amount of credits in the batch that have been
+  // locked up in escrow for use in the marketplace.
+  string escrowed_amount = 5;
```

Comment:
auto formatted

Reply:
Can we finish the proto lint pull request and make the formatting changes there (#721)?

Some of the files updated in this pull request are not related to the proposed changes (e.g. basket proto files). Also, the space before the colon does not appear to be a standard proto style guideline, is this something you added?

Reply:
> there is no space before the colon

You are adding spaces before the colon. Maybe that's what you meant to say.

```
primary_key : {fields : "id", auto_increment : true}
```

If this is the format we want to use, we should make sure this is consistent for everyone pushing code (#721). This is new to everyone else, so would be nice to make sure we are all aligned and using the same style.